dataprofiler/profilers/unstructured_text_profile.py

```diff
@@ -28,6 +28,9 @@ def __init__(self, name, options=None):
         # TODO: Add line length
         #self.line_length = {'max': None, 'min': None,...} #numeric stats mixin?
 
+        if options:
+            options.validate()
+
```

Comment:
i dont believe we do this in any other profile. However, we should validate that the options coming in our in fact `TextProfilerOptions`

Reply:
the validate is happening at the top level of profiler

Reply:
we can put it to UnstructuredProfiler later when we refactor all of this. For now, it's in the constructor.

Reply:
as this is in a feature branch, i don't think it is necessary to do this. I suggest removing.

Reply:
should do the same options check the other classes do

Reply:
e.g.
```python
if options and not isinstance(options, FloatOptions):
    raise ValueError("FloatColumn parameter 'options' must be of type"
                     " FloatOptions.")
```